src/actions/index.js

```diff
@@ -82,3 +83,10 @@ export const handleCurrentPage = (currentPage) => {
     currentPage,
   };
 };
+
+export const filterTodo = (filterState) => {
+  return {
+    type: HANDLE_FILTER,
+    filterState,
```

Comment:
see https://redux.js.org/tutorials/essentials/part-4-using-data#preparing-action-payloads

```suggestion
    payload: filterState,
```